client/src/partials/fiscal/fiscal.list.html

```diff
@@ -0,0 +1,83 @@
+<div class="flex-util" style="text-align:center">
+  <div class="container">
+    <strong>{{"FISCAL.TITLE" | translate }}</strong>  
+  </div>  
+</div>
+
+<div class="container-fluid">
+  <div class="container text-center">
+    <table style="width:75%; margin:auto" class="table table-condensed">
+      <thead>
+        <tr>
+          <th style="width:33%">
+            <a href uib-dropdown-toggle ng-click="FiscalCtrl.sort({by : 'label', order: FiscalCtrl.sortByName})">
+              {{"FORM.LABELS.NAME" | translate }} <span ng-if="FiscalCtrl.sorted === 'label'" class="glyphicon glyphicon-menu-{{FiscalCtrl.nameDesc}}"></span>
+            </a>
+          </th>
+
+          <th style="width:34%">
+            <a href uib-dropdown-toggle ng-click="FiscalCtrl.sort({by : 'created_at', order: FiscalCtrl.sortByDateCreated})">
+              {{"FORM.LABELS.DATE_CREATED" | translate }} <span ng-if="FiscalCtrl.sorted === 'created_at'" class="glyphicon glyphicon-menu-{{FiscalCtrl.dateDesc}}"></span>
+            </a>
+          </th>
+          <th style="width:33%">
+            <a href uib-dropdown-toggle ng-click="FiscalCtrl.sort({by : 'number_of_months', order: FiscalCtrl.sortByNumbrerOfMonth})">
+              {{"FORM.LABELS.NB_MONTH" | translate }} <span ng-if="FiscalCtrl.sorted === 'number_of_months'" class="glyphicon glyphicon-menu-{{FiscalCtrl.nbMonthDesc}}"></span>
+            </a>  
+          </th>
+        </tr>
+      </thead>
+    </table>
+
+    <div ng-repeat="fiscal in FiscalCtrl.fiscalYears | limitTo : FiscalCtrl.pageSize : (FiscalCtrl.currentPage - 1)*FiscalCtrl.pageSize ">
```

Comment:
In the future, I think adding `track by fiscal.id` might optimize this rendering.  But I am not sure how the implementation would look at the moment.